src/bundles/files/actions.js

```diff
@@ -214,7 +214,7 @@ const actions = () => ({
           ...fileFromStats({ ...stats, path }),
           fetched: time,
           type: 'file',
-          read: () => ipfs.cat(stats.cid),
+          read: (offset, length) => ipfs.cat(stats.cid, { offset, length }),
```

Comment:
nit: this CI action is failing but this is not a TS file.. we could add jsdoc.. 

do we want to disable typecheck CI while we are not full typescript?

Reply:
Let's keep it, we get contributions from people, and better if they include types (easier for us to review).
I've added jsdoc.